contracts/zapper-contract/SnowglobeZapperTraderJoe.sol

```diff
@@ -45,12 +35,15 @@ contract SnowglobeZapAvaxTraderJoe is ZapperBase {
         (uint256 reserveA, uint256 reserveB, ) = pair.getReserves();
         require(reserveA > minimumAmount && reserveB > minimumAmount, "Liquidity pair reserves too low");
 
-        bool isInputA = pair.token0() == tokenIn;
-        require(isInputA || pair.token1() == tokenIn, "Input token not present in liquidity pair");
+        address token0 = pair.token0();         // first token in the pool
+        address token1 = pair.token1();         // second token in the pool 
```

Comment:
This is the only file where we create new variables for token0 and token1 instead of just using pair.token0(). Whichever way we decide, it would be beneficial for us to be consistent so we don't have multiple ways of doing the same thing syntactically.